common.js

```diff
@@ -34,7 +34,12 @@ function insertInfoPane() {
         // contact info
         html += '<div id="sidebar-contact">';
         html += '<h3>Contacts</h3>';
-        html += '<p>TSC Chair: <a href="mailto:ken.museth@gmail.com">ken.museth@gmail.com</a> </br> Developers: </br><a href="mailto:openvdb@gmail.com">openvdb@gmail.com</a> </br> Discussion Forum:</br><a href="http://www.openvdb.org/forum/">openvdb-forum@googlegroups.com</a></p>';
+        html += '<p>';
+        html += 'TSC Chair:</br><a href="mailto:ken.museth@wetafx.co.nz ">ken.museth@wetafx.co.nz </a> </br>';
```

Comment:
There appear to be two paragraph separator characters (U+2029) in this line, one after each `nz`, causing the address to render as `wetafx.co.nzâ€©` in my browser, at least.

Reply:
Very strange, it's completely hidden from the diff too. I just fixed this.